pkg/standalone/common.go

```diff
@@ -28,19 +29,20 @@ const (
 	defaultComponentsDirName = "components"
 )
 
-// GetDaprPath returns the dapr installation path.
+// GetDaprRuntimePath returns the dapr runtime installation path.
 // The order of precedence is:
-//  1. From --dapr-path command line flag
-//  2. From DAPR_PATH environment variable
-//  3. $HOME/.dapr
-func GetDaprPath(inputInstallPath string) (string, error) {
-	if inputInstallPath != "" {
-		return inputInstallPath, nil
+//  1. From --runtime-path command line flag appended with `.dapr`
+//  2. From DAPR_RUNTIME_PATH environment variable appended with `.dapr`
+//  3. default $HOME/.dapr
+func GetDaprRuntimePath(inputInstallPath string) (string, error) {
```

Comment:
```suggestion
func GetDaprRuntimePath(daprRuntimePath string) (string, error) {
```

Reply:
We can also mention in the function docs comment that input comes from the flag.